openid-4-verifiable-presentations-1_0.md

```diff
@@ -357,23 +357,52 @@ Content-Type: application/json
 
 ## Using `scope` Parameter to Request Verifiable Credential(s) {#request_scope}
 
-Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 scope values.
+Wallets MAY support requesting presentation of Verifiable Credentials using OAuth 2.0 `scope` values.
 
-Such a scope value MUST be an alias for a well-defined Presentation Definition that will be 
-referred to in the `presentation_submission` response parameter. 
+In the case where all the Input Descriptors defined within the Presentation Definition are required within the request, the value contained in the scope parameter MUST match to the Presentation Definition id value. In the case where only a subset of Verifiable Credentials is required, each value contained in the scope parameter MUST match to one Input Descriptor(s) `id` value, as defined within the Presentation Definition.
 
-The specific scope values, and the mapping between a certain scope value and the respective 
-Presentation Definition is out of scope of this specification. 
+Since the values contained in the `scope` parameter are expressed as a list of space-delimited and case-sensitive strings, according to [@RFC6749], the Verifier that supports the `scope` parameter to request Verifiable Credential(s) MUST NOT include spaces in the Input Descriptors `id` values intended to be used within the `scope` parameter. It is then RECOMMENDED to use collision-resistant scopes values.
```

Comment:
```suggestion
Since the values contained in the `scope` parameter are expressed as a list of space-delimited and case-sensitive strings as defined in [@RFC6749], `id` values of the Input Descriptors and Presentation Definitions used as the `scope` parameter to request Verifiable Credential(s) MUST NOT include spaces. 

It is RECOMMENDED to use collision-resistant `scope` values.
```
if verifier supports `scope` parameter to request VCs, any input descriptor `id` can be used as a scope, no?